src/ScClient.ts

```diff
@@ -122,21 +128,31 @@ export class ScClient {
     sendData();
   }
 
+  private formSendAnswer(resolve: (arg: any) => void, reject: (arg: any) => void, success: any, errors?: any) {
```

Comment:
Will errors always be present in response? If so '?' can be removed

Reply:
Errors always present in response. If there are no than errors field is empty